packages/docs/site/docs/main/guides/for-theme-developers.md

```diff
@@ -4,4 +4,225 @@ slug: /for-theme-developers
+
+```json
+"steps": [
+	...,
+	{
+		"step": "writeFile",
+		"path": "/wordpress/wp-content/mu-plugins/rewrite.php",
+		"data": "<?php /* Use pretty permalinks */ add_action( 'after_setup_theme', function() { global $wp_rewrite; $wp_rewrite->set_permalink_structure('/%postname%/'); $wp_rewrite->flush_rules(); } );"
+	},
+	...
+]
+```
+
+### `updateUserMeta`
+
+With [`updateUserMeta`](/blueprints/steps#updateUserMeta) step, you can update any user metadata. For example, you could update the metadata of the default `admin` user of any WordPress installation:
+
+```json
+"steps": [
+	...,
+	{
+		"step": "updateUserMeta",
+		"meta": {
+			"first_name": "John",
+			"last_name": "Doe",
+			"admin_color": "modern"
+		},
+		"userId": 1
+	},
+	...
+]
+```
+
+### `importWxr`
+
+With [`importWxr`](/blueprints/steps#importWxr) step, you can import your own content via a `.xml` file previously [exported from an existing WordPress installation](https://wordpress.org/documentation/article/tools-export-screen/):
+
+```json
+"steps": [
+	...,
+	{
+		"step": "importWxr",
+		"file": {
+			"resource": "url",
+			"url": "https://raw.githubusercontent.com/wordpress-juanmaguitar/blueprints/install-theme-directory-from-repo/blueprints/install-activate-setup-theme-from-gh-repo/blueprint-content.xml"
+		}
+	},
+	...
+]
+```
+
+It is recommended that the exported `.xml` is uploaded to GitHub repository in the same directory the `blueprint.json` is
+
+:::tip
+One of the things you'll want to import for your default content are images. A recommended approach is to upload the images to your GitHub repo and search/replace the path for them in the exported `.xml` file using the URL format: `https://raw.githubusercontent.com/{repo}/{branch}/{image_path}`
+
+```html
+<!-- wp:image {"lightbox":{"enabled":false},"id":4751,"width":"78px","sizeSlug":"full","linkDestination":"none","align":"center","className":"no-border"} -->
+<figure class="wp-block-image aligncenter size-full is-resized no-border">
+	<img src="https://raw.githubusercontent.com/WordPress/blueprints/trunk/blueprints/install-activate-setup-theme-from-gh-repo/images/avatars.png" alt="" class="wp-image-4751" style="width:78px" />
+</figure>
+<!-- /wp:image -->
+```
+
+:::
+
+### `setSiteOptions`
+
+With [`setSiteOptions`](/blueprints/steps#setSiteOptions) step, you can set [site options](https://developer.wordpress.org/apis/options/#available-options-by-category) such as site name, description or which page to use for posts.
+
+```json
+"steps": [
+	...,
+	{
+		"step": "setSiteOptions",
+		"options": {
+			"blogname": "Rich Tabor",
+			"blogdescription": "Multidisciplinary maker specializing in the intersection of product, design and engineering. Making WordPress.",
+			"show_on_front": "page",
+			"page_on_front": 6,
+			"page_for_posts": 2
+		}
+	},
+	...
+]
+```
+
+There's also a shorthand [`siteOptions`](/blueprints/steps/shorthands#siteoptions) than can be used instead of the `setSiteOptions` step.
```

Comment:
"a shorthand `siteOptions` ___ that can be used..."

Should "step" be there in the blank? Or, should it be written "`siteOptions` shorthand"?

Reply:
Yes, I think this sounds better

> There's also a [`siteOptions`](/blueprints/steps/shorthands#siteoptions) shorthand that can be used instead of the `setSiteOptions` step.